portfolio/src/main/webapp/script.js

```diff
@@ -28,3 +28,9 @@ function addRandomQuote() {
   const quoteContainer = document.getElementById('quote-container');
   quoteContainer.innerText = quote;
 }
+
+function getRandomQuoteUsingArrowFunctions() {
```

Comment:
This is an optional comment, but, if you want a challenge, think about how to handle edge cases.

See comment that I left on Anthony's pull request:

https://github.com/anthonyqzhu/step/pull/9/files/7d4ef82d31855aa31b5e502a2b064a0f72d11a43#r433495655

Reply:
I'll look into it soon - thanks Jenny!!!